README.md

```diff
@@ -94,8 +112,8 @@ Example (using the sample table from the Query Configurator case):
 SELECT sensor_id, temperature, registered_at, location FROM test.test WHERE sensor_id IN (99051fe9-6a9c-46c2-b949-38ef78858dd1, 99051fe9-6a9c-46c2-b949-38ef78858dd0) AND registered_at > $__timeFrom and registered_at < $__timeTo
 ```
 
-1. Order of fields in the SELECT expression doesn't matter except `ID` field. This field used to distinguish different time series, so it is important to keep it on the first position.
-* **Identifier** - the first property in the SELECT expression must be the ID, something that uniquely identifies the data (e.g. `sensor_id`)
+1. Order of fields in the SELECT expression doesn't matter except `ID` field. This field used to distinguish different time series, so it is important to keep it or any other column with low cardinality on the first position.
+* **Identifier** - the first property in the SELECT expression should be the ID, something that uniquely identifies the data (e.g. `sensor_id`)
 * **Value** - There should be at least one numeric value among returned fields, if query result will be used to draw graph.
```

Comment:
Maybe we should suggest users to use type casting to reach that? For experienced C* users it must be obvious but for less seasoned people...

Reply:
But why? Casting is not necessary now because any numeric type(integers, floats) will suffice without casting.

Reply:
Yep, but what if they, for whatever reason, store it as a non-numeric type? I'm just trying to help those unlucky devils :)

Reply:
You mean text fields? Well, it wouldn't work - there is no way to cast text to numeric types afaik. I am not sure what is possible to clarify here. Do you have an example?

Reply:
You are right, it's too much of the edge case (and not our problem to deal with, tbh)